src/ReleaseHistory.md

```diff
@@ -1,9 +1,11 @@
 # BinSkim Release History
 
+* **BREAKING** Change from self-contained to dotnettool. [#306](https://github.com/microsoft/binskim/pull/306)

```

Comment:
>BREAKING [](start = 4, length = 8)

## **v1.7.0** [NuGet Package](https://www.nuget.org/packages/Microsoft.CodeAnalysis.BinSkim/1.7.0)

Reply:
instead of adding the version 1.7.1, I change to `Unreleased`, since people were thinking that we already released the `1.7.0` version.